client/src/Component/MeetingComponent/MeetingComponent.js

```diff
@@ -0,0 +1,201 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import '../MeetingComponent/MeetingComponent.css';
+import Icon from '@material-ui/core/Icon';
+import { makeStyles } from "@material-ui/core/styles";
+import Button from '@material-ui/core/Button';
+import { createMuiTheme } from '@material-ui/core/styles';
+import { grey, orange } from '@material-ui/core/colors';
+import { ThemeProvider } from '@material-ui/styles';
+import Paper from "@material-ui/core/Paper";
+import shadows from '@material-ui/core/styles/shadows';
+import Avatar from '@material-ui/core/Avatar';
+import Typography from '@material-ui/core/Typography';
+import GettingStartedButton from '../GettingStartedButton/GettingStartedButton';
+
+
+const outerTheme = createMuiTheme({
+  palette: {
+    secondary: {
+      main: orange[500],
+    },
+    primary:{
+      main: grey[600]
+    }
+  },
+});
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+   flexGrow: 1
+    
+  },
+  paper: {
+    // padding: theme.spacing(0),
+    // textAlign: "center",
+    // color: "red",
+        background: theme.palette.text.default,
+        boxShadow:shadows[3],
+       paddingBottom: theme.spacing(3),  
+  },
+  profile: {
+    padding: theme.spacing(3),
+    
+  }
+  
+}));
+
+export default function Height() {
```

Comment:
ideally we change this so that instead of predefined boxes it's done dynamically with something like `meetings.map`. For now you can predefine those meetings just in a variable so you can pretend you're getting data from the backend